rpcs3/Emu/Memory/vm.cpp

```diff
@@ -479,12 +500,21 @@ namespace vm
 		, size(size)
 		, flags(flags)
 	{
-		// Allocate compressed reservation info area (avoid RSX and SPU areas)
-		if (addr != 0xc0000000 && addr != 0xe0000000)
+		// Allocate compressed reservation info area (avoid SPU MMIO area)
+		if (addr != 0xe0000000)
 		{
 			utils::memory_commit(g_reservations + addr / 16, size / 16);
 			utils::memory_commit(g_reservations2 + addr / 16, size / 16);
 		}
+		else
+		{
+			// RawSPU LS
+			for (u32 i = 0; i < 6; i++)
+			{
+				utils::memory_commit(g_reservations + addr / 16 + i * 0x10000, 0x4000);
```

Comment:
I think it needs hw testing, checking if making a reservation is allowed on raw spu ls can be done with sys_process_is_spu_lock_line_reservation_address syscall.

Reply:
also.. there are only 5 raw spus at max